pkg/httpapi/api.go

```diff
@@ -287,12 +295,71 @@ func (a *APIRouter) GetApplicationDetails(w http.ResponseWriter, r *http.Request
 		Revision: revision,
 	}
 
+	var appServices = make([]envHealthResource, 0)
```

Comment:
would it be a good idea to use a map rather than creating separate variables for each type? 
something like `map[k8sType][]envHealthResource`!

Reply:
II converted it to use the map